src/helpers/getMovieUtils.ts

```diff
@@ -0,0 +1,12 @@
+const getMoviesByName = async (name: string) => {
```

Comment:
Could this file return the MovieResultData or MovieDetailData types explicitly so we know exactly what we are getting in the other files?